SSA/Projects/InstCombine/ForMathlib.lean

```diff
@@ -469,10 +469,15 @@ inductive Refinement {α : Type u} : Option α → Option α → Prop
   | bothSome {x y : α } : x = y → Refinement (some x) (some y)
   | noneAny {x? : Option α} : Refinement none x?
 
+@[simp]
 theorem Refinement.some_some {α : Type u} {x y : α} :
   Refinement (some x) (some y) ↔ x = y :=
   ⟨by intro h; cases h; assumption, Refinement.bothSome⟩
 
+@[simp]
+theorem Refinement.none_left :
+  Refinement none x? := .noneAny
+
```

Comment:
This this change needed?

In particular, the AliveStatements.lean file already uses Std.BitVec as well as `SSA/Projects/InstCombine/ForStd.lean`.

Reply:
The ForMathlib.lean file is completely unused. I deleted it to make clear it is really not needed anymore.